### PR TITLE
Clone the To Do List project

### PR DESCRIPTION
**Pull request by BenMukebo**
> ## Does my coding partner follow JS best practices?

- Create a new empty repo.
- Copy my code from the "To Do list" project and commit it as one big chunk of code in a feature branch.
- Open a pull request.

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+  font-size: inherit;
+  color: var(--primary);
+  cursor: pointer;
+}
+
+/* .tasks-list li label.compeleted {
+  text-decoration: line-through;
+  font-style: italic;
+  color: var(--secondary);
+} */
```
Comment (Whoistolu) [inline src/style.css]: - [ ] I feel it will be okay  to remove all unused lines of code `comments` in order to have a lean codebase

**Review by Whoistolu — COMMENTED** [review]
## STATUS: CHANGES REQUESTED ♻️ 

Hi @BenMukebo, You did a great job so far 👏🏼 👏🏼

However,  there are some issues that you still need to work on but you are almost there!

## Required Changes:
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Review by Whoistolu — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED ♻️ 

Hi @BenMukebo, You did a great job so far 👏🏼 👏🏼

However,  there are some issues that you still need to work on but you are almost there!

## Required Changes:
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] It will be nice to make your README file descriptive as it is a best practice.

**Review by Whoistolu — APPROVED** [review]
<h1>Status: Approve 🥇 ✔</h1>
Hi @BenMukebo , 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/style.css (hunk 1 above)
Comment (BenMukebo) [reply src/style.css]: Hi @Whoistolu, Thanks for your feedback 👏🏻

**Pull request merged**